web/src/pages/project/[projectId]/settings/integrations/posthog.tsx

```diff
@@ -46,7 +46,7 @@ export default function PosthogIntegrationSettings() {
   return (
     <div className="md:container">
       <Header
-        title="PostHog Integration"
+        title="PostHog Integration (Beta)"
```

Comment:
LOL

Reply:
IMO those can be two separate things. While the integration may still be Beta (workload, frequency, etc.) the event format can be 1.0.0

Reply:
agree, good point